8Knot/queries/pr_assignee_query.py

```diff
@@ -40,65 +35,45 @@ def pr_assignee_query(self, repos):
 
     query_string = f"""
                     SELECT
-                        *
+                        pa.pull_request_id,
+                        pa.id,
+                        -- below are timestamp not timestamptz
+                        pa.created,
+                        pa.closed,
+                        pa.assign_date,
+                        pa.assignment_action,
+                        left(pa.assignee::text, 15) as assignee
                     FROM
-                        explorer_pr_assignments pa
+                        augur_data.explorer_pr_assignments pa
                     WHERE
-                        pa.id in ({str(repos)[1:-1]})
+                        pa.id in %s
+                        and pa.created < now()
+                        and pa.closed < now()
+                        and pa.assign_date < now()
```

Comment:
Yeah the tiny corner-cases are acceptable. The major problem is wanting to avoid retrieving data that are mis-labelled as having occurred in the future